packages/cli/src/lib/commands/collect/options/awaitServeStdout.ts

```diff
@@ -1,18 +1,8 @@
-import { argv } from 'yargs';
-import { Param } from './awaitServeStdout.model';
-import { ArgvOption } from '../../../core/yargs/types';
+import { Options } from 'yargs';
 
-// inspired by: https://github.com/GoogleChrome/lighthouse-ci/blob/main/packages/cli/src/collect/collect.js#L28-L98
-export const param: Param = {
-  awaitServeStdout: {
-    alias: 'w',
-    type: 'string',
-    description: 'A string in stdou resulting from serving the app, to be awaited before start running the tests. e.g. "server running..."',
-    implies: ['w', 'serveCommand']
-  }
-};
-
-export function get(): string {
-  const { awaitServeStdout } = argv as any as ArgvOption<Param>;
-  return awaitServeStdout;
-}
```

Comment:
This is an anti pattern and must be complete  removed from the application! 

It will break when we migrate and it simply a bad way to manage state. 

We have decided to use a functional approach, so we will not use this getting mechanism!

It was also never used here